exonum/src/api/private/system.rs

```diff
@@ -186,8 +195,40 @@ impl Api for SystemApi {
             self_.ok_response(&::serde_json::to_value(info).unwrap())
         };
 
+        let self_ = self.clone();
+        let consensus_enabled_info = move |_: &mut Request| -> IronResult<Response> {
+            let info = self_.get_consensus_enabled_info();
+            self_.ok_response(&::serde_json::to_value(info).unwrap())
+        };
+
+        let self_ = self.clone();
+        let consensus_enabled_set = move |req: &mut Request| -> IronResult<Response> {
+            let map = req.get_ref::<Params>().unwrap();
```

Comment:
Do I understand correctly that the parameter is parsed from the URL rather than the body of the request? Is this intended? I'd say that using the body could be more flexible and is more commonplace for POST requests.

Reply:
`curl -H "Content-Type: application/json" -X POST -d '{"enabled":false}' http://127.0.0.1:8010/api/system/v1/consensus_enabled`

I've used this for tests, so no, I'm pretty sure that it uses the body for parameters.

Reply:
Judging from [docs](https://docs.rs/params/0.8.0/params/struct.Params.html), it seems that `Params` uses multiple sources available. Are you sure this is appropriate? It seems that the `bodyparser` crate could be used for more targeted parsing.

Reply:
^ @aleksuss

Reply:
Agree with @slowli. `bodyparser` crate is more appropriate. Take a look at this example:
https://github.com/exonum/cryptocurrency/blob/master/src/lib.rs#L276

Reply:
Ok. I'll create another issue+PR for this.